cpp/doxygen/Doxyfile

```diff
@@ -38,7 +38,7 @@ PROJECT_NAME           = libucxx
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = 0.37.00
+PROJECT_NUMBER         = $(UCXX_VERSION)
```

Comment:
Where does this get set?

Reply:
See `cpp/CMakeLists.txt` and `ci/build_docs.sh`.